### DISCUSSION TITLE
Have difficulty to find tutorial to build model for fracture initial and propagation for Brazilian Test

### DISCUSSION THREAD

**Discussion by miraboreasumr** [idaholab/moose#27766] [Q&A Modules: Phase field]
Could anyone show me some tutorials about modeling the fracture initial and propagation? I have studied module combine and phase field, but haven't progressed. Thank you.

Edit: I need to use Phase Field

**Comment by GiudGiud** [comment]
Hello

There's also documentation on fracture in xfem and peri dynamics if I am not mistaken

**Reply by miraboreasumr** [reply]
> Hello
> 
> There's also documentation on fracture in xfem and peri dynamics if I am not mistaken

Thanks for the reply, I forgot to mention that I need to sue Phase Field
